cmd/sql-main.go

```diff
@@ -37,6 +43,30 @@ var (
 			Name:  "recursive, r",
 			Usage: "sql query recursively",
 		},
+		cli.StringFlag{
+			Name:  "csv-input",
+			Usage: "csv input serialization option",
+		},
+		cli.StringFlag{
+			Name:  "json-input",
+			Usage: "json input serialization option",
+		},
+		cli.StringFlag{
+			Name:  "compression",
+			Usage: "input compression type",
+		},
+		cli.StringFlag{
+			Name:  "csv-output",
+			Usage: "csv output serialization option",
+		},
+		cli.StringFlag{
+			Name:  "csv-output-header,oh",
```

Comment:
I don't think we need the short form `oh` it is weird to pronounce and sounds like two options crammed together.